udata/core/issues/api.py

```diff
@@ -110,20 +85,46 @@ def post(self, id):
             on_issue_closed.send(issue, message=message)
         else:
             on_new_issue_comment.send(issue, message=message)
-        return marshal(issue, issue_fields), 200
+        return issue
 
 
 @ns.route('/', endpoint='all_issues')
```

Comment:
Same here: now there is only one endpoint for the collection, we should use `api.issues`